examples/getstarted/src/api/kitchensink/content-types/kitchensink/schema.json

```diff
@@ -130,6 +130,10 @@
     "morph_to_many": {
       "type": "relation",
       "relation": "morphToMany"
+    },
+    "custom_field": {
+      "type": "customField",
+      "customField": "plugin::mycustomfields.color"
```

Comment:
Should we commit this? I think we will remove it once we can save a registered customField to a contentType right?

Reply:
I thought it could be nice to keep it since I believe the goal of the kitchen sink content type is to show all attribute types